### PR TITLE
Fix pagination for lazy loading

### PR DESCRIPTION
**Pull request by crissi**

### CODE REVIEW

**Comment by mfn** [comment]
🎉

File: src/GraphQL.php [hunk 1]
```diff
@@ -332,11 +332,13 @@ protected function clearTypeInstances(): void
 
     public function paginate(string $typeName, string $customName = null): Type
     {
-        $name = $customName ?: $typeName.'_pagination';
+        $name = $customName ?: $typeName.'Pagination';
```
Comment (mfn) [inline src/GraphQL.php]: Could this have any kind of compatibility impact?

Comment (mfn) [reply src/GraphQL.php]: And, curious, I compared this to your other PR and unless I made a mistake, that's the only difference I could see. Why does that fix it? 🤷‍♀️

Comment (crissi) [reply src/GraphQL.php]: take a look in the PaginationType, it is the same way the name is constructed

Comment (crissi) [reply src/GraphQL.php]: > Could this have any kind of compatibility impact?

I don't think so

**Pull request merged**